pkg/aspect/listsManager.go

```diff
@@ -66,14 +67,20 @@ func (listsManager) Kind() config.Kind {
 
 func (listsManager) DefaultConfig() config.AspectParams {
 	return &aconfig.ListsParams{
-		CheckExpression: "src.ip",
+		CheckExpression: "source.ip",
 	}
 }
 
-func (listsManager) ValidateConfig(c config.AspectParams, _ expr.Validator, _ descriptor.Finder) (ce *adapter.ConfigErrors) {
-	lc := c.(*aconfig.ListsParams)
-	if lc.CheckExpression == "" {
-		ce = ce.Appendf("CheckExpression", "Missing")
+func (listsManager) ValidateConfig(c config.AspectParams, v expr.Validator, df descriptor.Finder) (ce *adapter.ConfigErrors) {
+	cfg := c.(*aconfig.ListsParams)
+	if cfg.CheckExpression == "" {
+		ce = ce.Appendf("CheckExpression", "no expression provided")
+		return
+	}
+	if t, err := v.TypeCheck(cfg.CheckExpression, df); err != nil {
```

Comment:
Chatted offline, we'll update the expression validator interface to add the new method in a follow on PR.